pkg/controller/constraint/stats_reporter.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	totalConstraintsName = "total_constraints"
+	totalConstraintsName = "constraints"
```

Comment:
update `totalConstraintsName` to match `violationsName`

Reply:
I'm wondering if `constraintsName` would be confusing. It might overlap with something like the name of the constraint. Is there some semantic name we could use, rather than trying to match the contents of the string?

Reply:
maybe add a `metric` in the name so it's differentiated. `constraintsMetricName` or `totalConstraintsMetricName` perhaps? i think it's fine to have `total` in the name as long as its consistent.

Reply:
Changed all "metric name"-type constants to have the `MetricName` suffix